.phpcs.xml

```diff
@@ -32,6 +32,9 @@
   <config name="minimum_supported_wp_version" value="5.8" />
 
   <rule ref="WordPress.NamingConventions.PrefixAllGlobals">
+    <properties>
+			<property name="prefixes" type="array" value="wp_component_library,wpcl" />
```

Comment:
This gets set at runtime in the composer run config. If it's helpful to have it here for IDE integration etc, that's fine, but we should remove it from being inlined as part of `composer.json` to avoid repeating ourselves.